### DISCUSSION TITLE
css file is not generated

### DISCUSSION THREAD

**Discussion by Kixo** [rainbow-me/rainbowkit#375] [Feedback]
Failed to compile.

./src/Providers.js
Module not found: Can't resolve '@rainbow-me/rainbowkit/styles.css' in '/home/xxxxx/xxxxxx/src'

**Comment by nickbytes** [comment]
Can you provide more details or a minimal reproducible example @Kixo ?

**Comment by hfhunter7** [comment]
@Kixo same issue for me @next v10

**Comment by markdalgleish** [comment]
My guess is you're using an older build tool that doesn't understand the `exports` field in `package.json` for ESM packages.

This is what ours looks like — note the aliasing of our CSS file: https://github.com/rainbow-me/rainbowkit/blob/7b9a1dffb54a971ef8c2c3029814df95f614360a/packages/rainbowkit/package.json#L9-L12

As a workaround, you should be able to import the CSS without an alias:

```ts
import '@rainbow-me/rainbowkit/dist/index.css';
```

**Comment by AndreiBelokopytov** [comment]
I faced with the same problem and my solution is to import styles into css file using the following syntax:
```
@import "~@rainbow-me/rainbowkit/styles.css";
```

**Comment by MateuszGroth** [comment]
Adding a `sideEffects: true` next to the css loader should do the job

```js
        {
          test: /\.css$/,
          use: [ MiniCssExtractPlugin.loader, "css-loader"],
          sideEffects: true, // this
        },
```
